hardware/data_acquisition_units.py

```diff
@@ -50,8 +72,27 @@ def __init__(self, device_name=None, max_voltage=None, rate=None):
         self.max_voltage = max_voltage
         self.task = None
 
+    #define two different read functions, one async and one sync
+    """
+    async def read(self, seconds, rate=None, max_voltage=None, timeout=0,
+             verbose=False, oversampling_ratio=10, task_name=""):
+        rate = self._get_rate(rate, oversampling_ratio)
+        max_voltage = self._get_max_voltage(max_voltage)
+
+
+        self.task = self.FOG_DAQ_Task(
+            self, seconds, rate, oversampling_ratio, max_voltage, asynchronous)
+
+        self.queue = asyncio.Queue()
+        # self.task.StartTask()
+        # while True:
+        #     yield (await self.queue.get())
+
+
+    """
+
     def read(self, seconds, rate=None, max_voltage=None, timeout=0,
-             verbose=False, oversampling_ratio=10, task_name="", 

```

Comment:
I see a lot of cases where a whitespace is removed. If Jon approved this, then go ahead. If not, be cautious of these changes as they add noise to the code-review and increase the potential for merge conflicts. 

Another reason to install a linter.